app/src/main/java/org/eyeseetea/malariacare/data/NetworkStrategy.java

```diff
@@ -0,0 +1,5 @@
+package org.eyeseetea.malariacare.data;
+
+public enum  NetworkStrategy {
```

Comment:
NetworkStrategy should be in the domain layer. Now this creates a dependency with the data layer from UserAccountUseCase